<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SearchSNP</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/dna.png</normaloff>icon/dna.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>751</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Search SNP</string>
    </property>
    <widget class="QTextEdit" name="inputSNP_display">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="browse">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="filename">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="search">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>440</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_chromosome">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Chromosome</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolchromosome">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="Chromo_display">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lb_position">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolposition">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>120</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="Posi_display">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>150</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lb_genechip">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>220</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>GeneChip</string>
     </property>
    </widget>
    <widget class="QComboBox" name="GeneChipBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>250</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nsp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sty</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lb_geneid">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>GeneID</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="GeneID_display">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>50</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_genesymbol">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>120</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>GeneSymbol</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="genesym_display">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>150</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lb_distance">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>220</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Distance</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tooldistance">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>220</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="dist_display">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>250</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lb_relationship">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>320</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Relationship</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolrelationship">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>320</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="rel_display">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>350</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lb_disease">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Disease</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tooldisease">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>20</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="dise_display">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>50</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>120</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Source_Website</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sourcewebBox">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>150</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Huge</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Kegg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pathway</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
